Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml

```
@@ -38,7 +38,7 @@
     tags:
     - Katana
   - type: Sprite
-    sprite: Objects/Weapons/Melee/katana.rsi
+    sprite: DeltaV/Objects/Weapons/Melee/katana.rsi
```

```suggestion
    sprite: DeltaV/Objects/Weapons/Melee/katana.rsi #DeltaV
```
Missing a comment on an upstream file.